.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to pyccel
+
+We welcome any and all contributions to pyccel!
+
+The following is a summary of the best ways to interact/help with this project.
+
+## Asking questions
+
+For simple questions (e.g. installation problems), we encourage you to use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/q-a).
+
+For more involved questions we also have a [slack channel](pyccel.slack.com).
+
+## Requesting features
+
+To request a new feature, please use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/ideas).
+This allows us to discuss the best implementation in the target languages before creating an issue to work on.
+
+## Reporting issues
+
+When reporting issues please include as much detail as possible about your
+operating system, and python version. Whenever possible, please
+also include a brief, self-contained code example that demonstrates the problem.
+
+## Ways to help
+
+There are many ways to help with the pyccel project which are more or less involved.
```

Comment:
I think the primary meaning of contributing is helping, so I suggest merging this part with the introduction.

Reply:
Fixed [a971b8b](https://github.com/pyccel/pyccel/pull/1252/commits/a971b8b5b296253d5a0fe2d5155fbf75b117fdaa)